schema.sql

```diff
@@ -41,25 +41,21 @@ ALTER TABLE animals ADD COLUMN owner_id INT,
   REFERENCES owners (id);
 
 /*Vet clinic database: add "join table" for visits*/
-
 CREATE TABLE vets(
 id INT GENERATED ALWAYS AS IDENTITY,
 name VARCHAR(55),
 age INT,
-date_of_graduation DATE
-);
+date_of_graduation DATE);
 
 CREATE TABLE specializations (
     species_id INT,
     vets_id INT,
   CONSTRAINT fk_species FOREIGN KEY(species_id) REFERENCES species(id) ON DELETE CASCADE,
-  CONSTRAINT fk_vets FOREIGN KEY(vets_id) REFERENCES vets(id) ON DELETE CASCADE
-);
+  CONSTRAINT fk_vets FOEIGN KEY(vets_id) REFERENCES vets(id) ON DELETE CASCADE);
 
 CREATE TABLE visits (
     animals_id INT,
     vets_id INT,
   date DATE,
```

Comment:
- [ ] Here using `date` as the column is not recognized which date. Kindly use `visit_date DATE`.

Reply:
That is a great solution, thank you

Reply:
Fixed.